modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ddl/DdlCommandHandler.java

```diff
@@ -230,6 +230,10 @@ private CompletableFuture<Boolean> handleAlterZone(AlterZoneSetCommand cmd) {
             zoneCfgBuilder.dataNodesAutoAdjustScaleUp(cmd.dataNodesAutoAdjustScaleUp());
         }
 
+        if (cmd.replicas() != null) {
```

Comment:
What's that?

Reply:
I've added this for supporting altering replicas for a zone through SQL.